storybook/src/stories/UseDeepCompareEffect/API/constants.ts

```diff
@@ -0,0 +1,244 @@
+export const COUNTRY_CODES = {
```

Comment:
Might this data be better stored as a JSON file? There's less compile/runtime overhead and Vite is [able to import](https://vitejs.dev/guide/features.html#json) from JSON nicely.